File: 프로젝트/get_(email)/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;
import android.widget.Toast;

import retrofit2.Call;
import retrofit2.Callback;
import retrofit2.Response;
import retrofit2.Retrofit;

public class MainActivity extends AppCompatActivity {

   TextView textView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        textView = findViewById(R.id.hello);

        Retrofit client = new RetrofitClient().getInstance();
        Retrofitinterface retrofitinterface = client.create(Retrofitinterface.class);

        Call<User> call = retrofitinterface.getUser();
        call.enqueue(new Callback<User>() {
            @Override
            public void onResponse(Call<User> call, Response<User> response) {
                User result = response.body();
                textView.setText(result.getData().getEmail());
            }

            @Override
            public void onFailure(Call<User> call, Throwable t) {
                makeFailToast();
            }
        });


    }
    void makeFailToast(){
        Toast.makeText(this,"server connection failed", Toast.LENGTH_LONG).show();
    }
}

File: 프로젝트/GRAMO/app/src/main/java/com/example/gramo/MainActivity.kt
package com.example.gramo

import android.app.ProgressDialog.show
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.prolificinteractive.materialcalendarview.CalendarDay
import com.prolificinteractive.materialcalendarview.MaterialCalendarView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        calendarView.setOnDateChangedListener { materialCalendarView: MaterialCalendarView, calendarDay: CalendarDay, b: Boolean ->
            val bottomDialogFragment: BottomDialogFragment = BottomDialogFragment {
                when (it){
                    0 ->
                }
            }
            bottomDialogFragment.show(supportFragmentManager, bottomDialogFragment.tag)
        }
    }
}